services/machine_learning/Dockerfile-dev

```diff
@@ -0,0 +1,17 @@
+# Use an official Python runtime as a parent image
+FROM python:3.6-slim
+
+# Set the working directory to /app
+WORKDIR /app
+
+# Copy the current directory contents into the container at /app
+ADD . /app
+
+# Install any needed packages specified in requirements.txt
+RUN pip install -r requirements.txt
+
+# Define environment variable
+ENV NAME World
```

Comment:
Is this used? I removed it when I merged your branch into mine and changed this file a bit